mycli/main.py

```diff
@@ -794,7 +794,7 @@ def configure_pager(self):
         else:
             self.explicit_pager = False
 
-        if cnf['skip-pager']:
+        if cnf['skip-pager'] or self.config['main'].as_bool('enable_pager'):
```

Comment:
Now that it is renamed `enable_pager`, should we use `not` in front of it? It looks like right now it will disable the pager if `enable_pager` is set to `True`.